service_test.go

```diff
@@ -289,6 +288,29 @@ func TestCreateService(t *testing.T) {
 	autopilot.Equals(t, 1, len(result.Aliases))
 }
 
+func TestUpdateService(t *testing.T) {
+	//Arrange
+	request := `{"query":
+		"mutation ServiceUpdate($input:ServiceUpdateInput!){serviceUpdate(input: $input){service{apiDocumentPath,description,framework,htmlUrl,id,aliases,language,lifecycle{alias,description,id,index,name},name,owner{alias,id},preferredApiDocument{id,htmlUrl,source{... on ApiDocIntegration{id,name,type},... on ServiceRepository{baseDirectory,displayName,id,repository{id,defaultAlias},service{id,aliases}}},timestamps{createdAt,updatedAt}},preferredApiDocumentSource,product,repos{edges{node{id,defaultAlias},serviceRepositories{baseDirectory,displayName,id,repository{id,defaultAlias},service{id,aliases}}},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount},tags{nodes{id,key,value},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount},tier{alias,description,id,index,name},timestamps{createdAt,updatedAt},tools{nodes{category,categoryAlias,displayName,environment,id,url,service{id,aliases}},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}},errors{message,path}}}",
+		"variables":{"input":{"id": "123456789"}}
+	}`
+	response := `{"data": {"serviceUpdate": {
+     "service": {{ template "service_1" }},
+     "errors": []
+ }}}`
+
+	client := ABetterTestClient(t, "service/update", request, response)
+
+	// Act
+	_, err := client.UpdateService(ol.ServiceUpdateInput{
```

Comment:
we should validate the response object has a correct value

Reply:
Oh yeah, missed that.  Just added one.